SimplenoteShare/ViewControllers/ShareViewController.swift

```diff
@@ -195,9 +224,17 @@ private extension ShareViewController {
     }
 }
 
+extension UIDevice {
+    static var isPad: Bool {
+        current.userInterfaceIdiom == .pad
+    }
+}
```

Comment:
Question, we already have this isPad check built into Simplenote in UIDevice+Simplenote.swift.  I added it here because that file's targets don't include Simplenote Share.
Could we simply add SimplenoteShare to the targets for that file?

Reply:
Actually....  I went back to try that/test it and this update isn't using the isPad bool anymore, so I can just remove it